### DISCUSSION TITLE
People closing unresolved issues

### DISCUSSION THREAD

**Discussion by mstevensm2** [Chia-Network/chia-blockchain#16795] [Support]
How can someone close an unresolved issue just because the original poster of the bug deleted everything and rebuilt everything from scratch when the actual issue still exists and is unresolved?

**Comment by BrandtH22** [comment]
Hey @mstevensm2 , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia). 

As for issues, do you have the issue number or a link to it? (all issues are still accessible on discord but you may need to remove the filter for showing open issues)

What was the original issue?

**Reply by mstevensm2** [reply]
Thanks for replying. It’s Issue #16697
He said he closed it because the original poster found a workaround, which was to delete everything and start over from scratch but the original issue is still unresolved. It seems strange that you close an issue when the issue is unresolved.

**Reply by BrandtH22** [reply]
Hey @mstevensm2 , I believe we were able to get you all setup in discord (https://discord.com/channels/1034523881404370984/1034870571864948777/1172004606423875606)

If this was not you then please reopen this ticket and I will further assist.

Recap of the solution for this issue:
1. Stop chia
2. Delete the height-to-hash and sub-epoch-summary files in the wallet DB folder ~\.chia\mainnet\db\. NOTE- also make sure the DB file is still the same size as when you moved it there. If it is not let me know as there are some more steps.
3. Start chia and give it upwards of 30 minutes to start services and begin syncing
